### ISSUE TITLE
Read code descriptions with a screen reader 

### ISSUE THREAD

**Issue by MesterPerfect** [microsoft/vscode#144934]
good morning 
I'm a blind user and I use Visual Studio Code to write my own HTML and Python projects
Visual Studio Code has a great feature which is information and description about any code that has been written
The information about any code can be shown by placing the mouse pointer over it 
![code](https://user-images.githubusercontent.com/44323153/157987180-11f28bd1-cce7-4329-8790-51a944ba6a52.jpg)

But the blind user cannot place the mouse pointer over the code
For this, can you add a quick shortcut that reads the description of the code
So that the blind user selects the code that he wants his description
After that, he presses on a specific shortcut, and the hint containing the description of the code will be shown
Thank you

**Comment by MesterPerfect** [comment]
@joyceerhl

**Comment by isidorn** [comment]
@MesterPerfect thanks for using VS Code and for opening this issue.
What you are searching for should already be possible. Just press `Ctrl + K, Ctrl + I` - that is a chord, so first press the first part then the second. This should open the hover and move focus to it so the screen readers reads the content.
Try it out and let us know how this works for you.

We are also looking forward to more feedback. Thanks!

**Comment by MesterPerfect** [comment]
not working for me
I already used shortcut ctrl+k and ctrl+i but nothing happened
thanks @isidorn

**Comment by isidorn** [comment]
Is the cursor on the right location? Can you share the content of the html file that you are tying this on and your cursor location? So I can try to reproduce on my machine. Thanks

**Comment by MesterPerfect** [comment]
> Is the cursor on the right location? Can you share the content of the html file that you are tying this on and your cursor location? So I can try to reproduce on my machine. Thanks

I am a blind user
I can't move cursor
So I need a way to read code description using keyboard

**Comment by isidorn** [comment]
@MesterPerfect I understand that, but I meant the editor cursor. That one moves where your keyboard focus is. You can move it using up / down / left / right.
So can you share your HTML file and the position of your keyboard cursor when you do this so I try to repro.

Thank you!

**Comment by MesterPerfect** [comment]
@isidorn
ok I moved cursor to line 3 where the <title> tag is located
After that I press shortcut ctrl+i because shortcut ctrl+k does nothing 
![2](https://user-images.githubusercontent.com/44323153/159602390-834184f3-9ed3-44b4-97af-5801e44f86fb.jpg)

**Comment by isidorn** [comment]
@MesterPerfect you need to press `ctrl + k`, and the `ctrl+i`. It is a key chord, they need to be pressed one after the other.

**Comment by MesterPerfect** [comment]
@isidorn
This really works
Thank you

**Comment by isidorn** [comment]
Glad to hear. Let me close this issue then.
And you can customise that keyboard shortcut if you do not like. Just open keyboard preference and search for command "editor show hover"
